source/includes/_dataset_security.md

```diff
@@ -0,0 +1,339 @@
+
+* `GET` [retrieve the default security ruleset](#retrieve-the-default-security-ruleset)
+* `PUT` [update the default security ruleset](#update-the-default-security-ruleset)
+* `DELETE` [reset the default security ruleset](#reset-the-default-security-ruleset)
+
+#### User level security ruleset
+
+`/dataset/{DATASET_UID}/security/users`
+
+* `GET` [Retrieve all user level security rulesets](#retrieve-all-user-level-security-rulesets)
+* `POST` [Create a new user level security ruleset](#create-a-new-user-level-security-ruleset)
+
+`/dataset/{DATASET_UID}/security/users/{USERNAME}`
+
+* `GET` [Retrieve a user level security ruleset](#retrieve-a-user-level-security-ruleset)
+* `PUT` [Update a user level security ruleset](#update-a-user-level-security-ruleset)
+* `DELETE` [Delete a user level security ruleset](#delete-a-user-level-security-ruleset)
+
+#### Group level security ruleset
+
+`/dataset/{DATASET_UID}/security/groups`
+
+* `GET` [Retrieve all group level security rulesets](#retrieve-all-group-level-security-rulesets)
+* `POST` [Create a new group level security ruleset](#create-a-new-group-level-security-ruleset)
+
+`/dataset/{DATASET_UID}/security/groups/{group_id}`
+
+* `GET` [Retrieve a group level security ruleset](#retrieve-a-group-level-security-ruleset)
+* `PUT` [Update a group level security ruleset](#update-a-group-level-security-ruleset)
+* `DELETE` [Delete a group level security ruleset](#delete-a-group-level-security-ruleset)
+
+## The general access policy
+
+A flag indicating whether the dataset is accessible by all people having access to the domain or just a few with explicit authorization.
+
+There are two acceptable values for the flag:
+
+* `domain`
+* `restricted`
+
+## Retrieve the general access policy
+
+> Definition
+
+```http
+GET https://{YOURDOMAIN}.opendatasoft.com/api/management/v2/datasets/{DATASET_UID}/security/access_policy
+```
+
+> Example request
+
+```shell
+curl https://yourdomain.opendatasoft.com/api/management/v2/datasets/da_XXXXXX/security/access_policy \
+    -u username:password
+```
+
+> Example response
+
+```json
+"domain"
+```
+
+Retrieves the general access policy set for the dataset.
+
+## Set the general access policy
+
+> Definition
+
+```http
+PUT https://{YOURDOMAIN}.opendatasoft.com/api/management/v2/datasets/{DATASET_UID}/security/access_policy
+```
+
+> Example request
+
+```shell
+curl https://yourdomain.opendatasoft.com/api/management/v2/datasets/da_XXXXXX/security/access_policy \
+    -u username:password \
+    -X PUT \
+    -d '"restricted"'
+```
+
+> Example response
+
+```json
+"restricted"
+```
+
+Sets the general access policy for the dataset.
+
+### Payload
+
+A single string. <br> Possible values are `domain` and `restricted`.
+
+## The ruleset object
+
+A set of rules defining what the target can see.
+
+The target can be:
+
+* a specific user
+* a specific group
+* anyone who has access to the portal (the default ruleset) and who's not affected by a user/group ruleset
```

Comment:
"anyone who has access to the portal (**through** the default ruleset) and who's not affected by a user **or a** group ruleset"